README.md

```diff
@@ -119,3 +119,20 @@ For more information about deployment to dev, check out DevOps' [excellent docum
 We use [OpenTelemetry](https://opentelemetry.io/) for tracing, so that every request has a tracing span that can 
 be viewed in [Google Cloud Trace](https://cloud.google.com/trace). (This is not yet fully set up here - to be done in TSPS-107). 
 See [this DSP blog post](https://broadworkbench.atlassian.net/wiki/x/AoGlrg) for more info.
+
+### Running the end-to-end tests
+
+The end-to-end test is specified in `.github/workflows/run-e2e-tests.yaml`. It calls the test script defined 
+in the dsp-reusable-workflows repo [here](https://github.com/broadinstitute/dsp-reusable-workflows/blob/main/e2e-test/tsps_e2e_test.py).
+
+The end-to-end test is run nightly on the dev environment. To test it on a specific development branch:
```

Comment:
would update this to `on a specific branch` just cuz developmeent could be confused for like the "dev" branch which isnt a thing but w/e.  Or use `feature branch` like you use in the next sentence.

Reply:
ah thanks, I meant feature branch!